CHANGELOG.md

```diff
@@ -1,47 +1,51 @@
 # Changelog
 
-## Unreleased
+## 4.5.0 (Feature release)
 
 ### New features
 
-#### Visually separate multiple summary lists on a single page with summary cards
+#### Use summary cards to visually separate multiple summary lists on a single page
 
-You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards) to help you design and build pages with multiple summary lists to help you visually distinguish them and to allow users to use actions that apply to entire lists.
+You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards). This new variant of the Summary list component can help you:
+
+design and build pages with multiple summary lists
+show visual dividers between summary lists
+allow users to apply actions to entire lists
 
 This was added in [pull request #2931: Add summary card enhancement to summary list](https://github.com/alphagov/govuk-frontend/pull/2931).
 
 #### Search within accordion content on supporting browsers
 
-We've updated the Accordion component to use the new [`hidden="until-found"` attribute value](https://developer.chrome.com/articles/hidden-until-found/).
+We've updated the Accordion component to use the new  [`hidden="until-found"` attribute value](https://developer.chrome.com/articles/hidden-until-found/).
 
 This allows the browser's native 'find in page' functionality to search within and automatically open sections of the accordion. Currently, this functionality is only supported by recent versions of Google Chrome, Microsoft Edge and Samsung Internet.
 
 This was added in pull requests:
 
-- [#3053: Enhance the Accordion component with `hidden='until-found'`](https://github.com/alphagov/govuk-frontend/pull/3053)
+- [#3053: Enhance the Accordion component with  `hidden='until-found'`](https://github.com/alphagov/govuk-frontend/pull/3053)
 - [#3095: Hide Accordion content (again) during `.js-enabled` page load](https://github.com/alphagov/govuk-frontend/pull/3095)
 
 #### Source maps for precompiled files
 
-You can now use [source maps](https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html) to help identify errors and console messages that come from GOV.UK Frontend precompiled files.
+You can now use [source maps](https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html) to help identify errors and consolidate messages from GOV.UK Frontend precompiled files.
```

Comment:
Mind if we keep "console messages"?

Happy to reword it but it's definitely messages from `console.log()` rather than "consolidate"

```suggestion
You can now use [source maps](https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html) to help identify errors and console messages from GOV.UK Frontend precompiled files.
```

Reply:
@claireashworth do you have any thoughts here?

Reply:
I completely missed this when I was parsing it, sorry!

Reply:
Have to admit, I read that wrong and thought it was a typo - 'console messages' as in 'messages that needed consoling' (silly me, but amusing).
Yes, let's leave it as 'console messages'.